src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -390,6 +390,8 @@ public override FlowAnalysisAnnotations ReturnTypeFlowAnalysisAnnotations
             }
         }
 
+        public override IImmutableSet<string> ReturnNotNullIfParameterNotNull => ImmutableHashSet<string>.Empty;
```

Comment:
So this won't work on property setters?

Reply:
Currently indexers are not supported. I'm looking into it. May add to this PR if easy. Otherwise will file a follow-up issue.
I don't think there are property scenarios except indexers.

---
In reply to: [303561556](https://github.com/dotnet/roslyn/pull/37201#discussion_r303561556) [](ancestors = 303561556)